pyodide-build/pyodide_build/buildpkg.py

```diff
@@ -450,7 +470,10 @@ def package_wheel(
     post = build_metadata.get("post")
     if post:
         bash_runner.env.update({"PKGDIR": str(pkg_root)})
-        bash_runner.run(post, check=True)
+        result = bash_runner.run(post)
+        if result.returncode != 0:
+            print("ERROR: post ({post}) failed")
```

Comment:
Is this supposed to be a format string here? Also, post is generally a multiline script so I don't know whether this would look good.

Reply:
Oops, was removing those, missed that one.